Automattic/wp-calypso
# Stats: Add an option to contact support for a classification update

Related to #80259

## Proposed Changes

* Add a mechanism to contact support and request an update of a site classification

## Testing Instructions



* due to missing site classification this can only be tested locally
* force your environment to classify sites as commercial (`isCommercial` variable in the main purchase component)
* go to stats and apply `stats/type-detection` feature flag
* trigger commercial purchase flow
* verify that the section at the bottom is visible like on the screenshot below
* remove the forced classification
* verify that the section at the bottom **is not** visible like on the screenshot below

<img width="942" alt="SCR-20230906-khfp" src="https://github.com/Automattic/wp-calypso/assets/112354940/a6dabb72-b19e-4a46-a7af-5dde696b30c4">

## Pre-merge Checklist
